### PR TITLE
Feature/behavior sticky

### PR DESCRIPTION
**Pull request by juanmaguitar**
- ⚙️ Task: https://jira.scmspain.com/browse/SUIC-4
- 📄 Paper: https://paper.dropbox.com/doc/SUI-Sticky-WIP-JgDUrAerXjVoxGq5TuotL
- 🌍 Demo Online: https://sui-components-feature-behavior-sticky.now.sh/workbench/behavior/sticky/demo

### CODE REVIEW

File: components/behavior/sticky/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# BehaviorSticky
+
+> Description
```
Comment (midudev) [inline components/behavior/sticky/README.md]: Can we have a description? :D

File: components/behavior/sticky/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@keyframes slide-down {
+  0% {
+    transform: translateY(-100%);
+  }
+  100% {
+    transform: translateY(0);
+  }
+}
+
+.isSticky {
```
Comment (midudev) [inline components/behavior/sticky/src/index.scss]: Is fine to have this className not inside the `sui-BehaviorSticky` one?

**Comment by midudev** [comment]
Sadly demo is not working. I guess that's because the scroll of the studio and the content is different. :(

**Review by midudev — DISMISSED** [review]

File: demo/behavior/sticky/demo/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@schibstedspain/sui-atom-button": "^1.23.0"
```
Comment (rmoralp) [inline demo/behavior/sticky/demo/package.json:13]: Fix version to 1

File: components/behavior/sticky/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# BehaviorSticky
+
+This component helps in the implementation of sticky components
```
Comment (rmoralp) [inline components/behavior/sticky/README.md:3]: Could you add a link to the github of library we use on it? In order to have better documentation about how it works
https://github.com/garthenweb/react-stickup#readme

File: components/behavior/sticky/package.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf ./lib && npx mkdirp ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "react-stickup": "^1.7.2"
```
Comment (jcger) [inline components/behavior/sticky/package.json]: please remove the ^

File: demo/behavior/sticky/demo/index.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+
+import React from 'react'
+// import cx from 'classnames'
```
Comment (jcger) [inline demo/behavior/sticky/demo/index.js]: please remove this and line 23

File: components/behavior/sticky/src/index.scss [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  0% {
+    transform: translateY(-100%);
+    opacity: 0;
+    z-index: -10000;
+  }
+
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+    z-index: 10000;
```
Comment (jcger) [inline components/behavior/sticky/src/index.scss]: this should be a sui-theme token

**Review by jcger — APPROVED** [review]

**Review by rmoralp — APPROVED** [review]

**Pull request merged**